eos/b-decays/b-to-3l-nu.cc

```diff
@@ -90,8 +90,8 @@ namespace eos
 
         /*!
          * differential decay width of 2 kinematic variables
-         * q2 is the invariant mass of the off-shell photon in the range 4 m_l'^2 <= q2 <= (m_B - k2)^2
-         * k2 is the invariant mass of the W-meson in the range m_l^2 <= k2 <= (m_B -q2)^2
+         * q2 is the invariant mass of the off-shell photon in the range 4 m_l'^2 <= q2 <= (m_B - sqrt(k2) )^2
```

Comment:
That does not reflect what we had written in [KKvDZ:2022A], which is: $4 m_{\ell'}^2 \leq q^2 \leq (m_B - m_\ell)^2$. $k^2$ seems to be correct.

Reply:
fixed